ui/src/shared/graphs/helpers.ts

```diff
@@ -187,27 +187,47 @@ export const getDataUUID = (
   dataType: DataType
 ): string => {
   if (dataType === DataType.influxQL) {
-    return getDeep(data, '0.response.uuid', '')
+    return getInfluxQLDataUUID(data as TimeSeriesServerResponse[])
   } else {
-    return getDeep(data, '0.id', '')
+    return getFluxDataUUID(data as FluxTable[])
   }
 }
 
+const getInfluxQLDataUUID = (data: TimeSeriesServerResponse[]) => {
```

Comment:
wanna annotate this with a `string` return type?

Reply:
👍